dartagnan/src/main/java/com/dat3m/dartagnan/Dartagnan.java

```diff
@@ -103,12 +108,16 @@ public static void main(String[] args) throws Exception {
                     options.getSMTSolver());
                  ProverEnvironment prover = ctx.newProverEnvironment(ProverOptions.GENERATE_MODELS))
             {
-                Result result;
+                Result result = Result.UNKNOWN;
+            	Preconditions.checkArgument(Arrays.asList(Analysis.values()).contains(options.getAnalysis()), 
```

Comment:
This is just a complex way to check for `NULL` (a value of enum type is always contained in `Enum.values` unless it is null).